Sources/TuistEnvKit/Installer/Installer.swift

```diff
@@ -180,6 +180,9 @@ final class Installer: Installing {
             }
             try FileHandler.shared.createFolder(installationDirectory)
 
+            try FileHandler.shared.copy(from: temporaryDirectory.path.appending(component: Constants.templatesDirectoryName),
```

Comment:
Similar comment here. If the user tries to install an old version of Tuist that doesn't contain the templates directory. This command will fail.